addons/account/wizard/account_move_reversal.py

```diff
@@ -47,14 +51,15 @@ def _compute_from_moves(self):
             record.move_type = move_ids.move_type if len(move_ids) == 1 else (any(move.move_type in ('in_invoice', 'out_invoice') for move in move_ids) and 'some_invoice' or False)
 
     def _prepare_default_reversal(self, move):
+        reverse_date = self.date if self.date and self.date_mode == 'custom' else move.date
```

Comment:
self.date is required so self.date is always True

You probably want to have the `date` field required only when `date_mode` is `custom`

Reply:
done